### ISSUE TITLE
Run serenityos.org on Serenity

### ISSUE THREAD

**Issue by boricj** [SerenityOS/serenity#792]
Based on the HTTP response headers, the website of serenityos.org runs on Ubuntu+Apache. A cool dogfooding scenario would be to run this website with Serenity.

**Comment by danboid** [comment]
Absolutely, but first we need stuff like a fully functional ifconfig, a network interface config file, DHCP support and of course a web server is going to help too.
